config/jobs/kubernetes-sigs/cluster-api/cluster-api-main-periodics-upgrades.yaml

```diff
@@ -52,7 +52,7 @@ periodics:
           cpu: 6000m
           memory: 6Gi
   annotations:
-    testgrid-dashboards: sig-cluster-lifecycle-cluster-api
+    testgrid-dashboards: sig-cluster-lifecycle-cluster-api-main
```

Comment:
Moved all main jobs into the `sig-cluster-lifecycle-cluster-api-main` dashboard. So the dashboard group and dashboard names are not conflicting

Reply:
SGTM, also indicates the branch name